DigitalLearningSolutions.Data/DataServices/GroupsDataService.cs

```diff
@@ -218,5 +209,50 @@ public void AddDelegateToGroup(int delegateId, int groupId, DateTime addedDate,
                 new { groupId, delegateId, addedDate, addedByFieldLink }
             );
         }
+
+        public Group? GetGroupAtCentreById(int groupId, int centreId)

```

Comment:
This method looks to be duplicating the entirety of the GetGroupsAtCentre method, but instead just retrieving a single group. That seems a little overkill to me.

Could we instead make a new GroupService method called GetGroupAtCentreById that calls the old GetGroupsAtCentre and then only returns the group with the supplied groupId?